x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -25,13 +25,26 @@ export interface ScoreTrendDoc {
   >;
 }
 
-export const getTrendsQuery = () => ({
+export type Trends = Array<{
+  timestamp: string;
+  summary: Stats;
+  clusters: Record<string, Stats>;
+}>;
+
+export const getTrendsQuery = (policyTemplate: PosturePolicyTemplate) => ({
   index: BENCHMARK_SCORE_INDEX_DEFAULT_NS,
   // large number that should be sufficient for 24 hours considering we write to the score index every 5 minutes
   size: 999,
   sort: '@timestamp:desc',
   query: {
     bool: {
+      filter: [
+        {
+          term: {
+            policy_template: policyTemplate,
+          },
+        },
+      ],
```

Comment:
<details>
<summary>super nit</summary>
so prettier can do compact mode too if we help it. clear some `\n` and it all fits nicely in a single row:
<code>
filter: [{ term: { policy_template: policyTemplate } }]
</code>
</details>